docs/pages-for-subheaders/aws-marketplace-payg-integration.md

```diff
@@ -0,0 +1,273 @@
+
+Once Rancher has been deployed from the listing on AWS and billing is active, there is no need to make a specific purchase for additional nodes. Billing is dynamic and based on the number of nodes Rancher is managing. Just deploy or on-board additional clusters to Rancher as needed.
+
+#### Is this an annual commitment, will it auto-renew?
+
+By default, the Rancher Prime listing in AWS is billed on a monthly cycle, based on usage. Billing is on-going for as long as Rancher is deployed.
+
+It is possible to set up an annual commitment via an AWS Private Offer, these will need to be reviewed and renewed at the end of the term, or the deployment will drop back to the default monthly billing cycle.
+
+### Technical (Billing)
+
+#### Do I need a Kubernetes cluster running in AWS to install Rancher and be billed via the AWS Marketplace?
+
+Yes. To benefit from monthly billing via the Marketplace, the primary Rancher cluster must be an EKS Cluster running in your AWS Account.
+
+#### On which Kubernetes distributions can the AWS Marketplace listing be deployed?
+
+The AWS Marketplace listing for Rancher Prime with Marketplace billing must be deployed on Amazon EKS. Downstream clusters / managed worker nodes can be running on any CNCF compliant Kubernetes platform, EKS, EKS-A, Rancher Kubernetes Engine, etc.
+
+#### What is the deployment mechanism?
+
+The AWS Marketplace listing for Rancher Prime is deployed using Helm.
+
+#### What is the easiest way to get started?
+
+One of the easiest ways to get started is to deploy the AWS Marketplace listing for Rancher Prime to an existing EKS cluster. Follow the instructions in the usage section, a Helm chart takes care of the installation and the setup for billing.
+
+#### What is the minimum version of Rancher required to support AWS Marketplace billing?
+
+The minimum version supporting marketplace billing is Rancher 2.7.9.
+
+#### What version of Rancher is installed when using the Marketplace listing?
+
+The AWS Marketplace listing for Rancher Prime is tied to a specific version of Rancher, typically the latest version available at the time of the listing update. Please check the listing for further information.
+
+#### I need a prior version of Rancher, can I still use the listing?
+
+No. There is no choice over the Rancher version when deploying using the AWS Marketplace listing. If a prior version of Rancher is required, this must be installed manually using the standard documentation. Billing via AWS Marketplace may not be supported with previous versions.
+
+#### How often is the listing updated (including the version of Rancher, etc.)?
+
+Cloud marketplace listings are tied to a specific version of Rancher, usually the latest
+version available at the time of listing.
+
+Typically, these are updated quarterly, or more frequently if there are security issues.
+
+To update Rancher to a current version before the marketplace listing is updated, please
```

Comment:
I dont think this is supported as it has the potential to break the billing adapter

Reply:
@jeremy-moffitt removing this sentence: "To update Rancher to a current version before the marketplace listing is updated, please see the product documentation."

b298c254cd1bb2f73cc92b1774d54b4f9d117c95